packages/xdl/src/detach/AssetBundle.js

```diff
@@ -46,21 +27,17 @@ export async function bundleAsync(
           extensionIndex >= 0
             ? asset.substring(prefixLength, extensionIndex)
             : asset.substring(prefixLength);
-        await saveUrlToPathAsync(
```

Comment:
Could you please make a note in the changelog that the next release expects SDK 25+?

Reply:
Done